bundles/org.openhab.binding.velbus/README.md

```diff
@@ -1,141 +1,266 @@
+| vmb1led | 1-channel 0-10V controlled PWM dimmer for led strips |
+| vmb1ry | 1-channel relay module |
+| vmb1ryno | 1-channel relay module with potential-free changeover contact |
+| vmb1rynos | 1-channel relay module with potential-free changeover contact |
+| vmb1rys | 1-channel relay module with input |
+| vmb1ts | Temperature Sensor Module |
+| vmb2bl | 2-channel blind control module |
+| vmb2ble | 2-channel blind control module with extended possibilities |
+| vmb2pbn | Push-button interface for Niko 1- or 2-fold push-buttons |
+| vmb4an | Analog I/O module |
+| vmb4dc | 4-channel 0/1-10V dimmer controller |
+| vmb4ry | 4-channel relay module |
+| vmb4ryld | 4-channel relay module with voltage outputs |
+| vmb4ryno | 4-channel relay module with potential-free contacts |
+| vmb6in | 6-channel input module |
+| vmb6pbn | Push-button interface module for Niko 4- or 6-fold push-button |
+| vmb7in | 7-channel input module (potentialfree + pulse) |
+| vmb8ir | Infrared remote control receiver module |
+| vmb8pb | 8-Channel Push Button module |
+| vmb8pbu | Push-button interface with 8 channels for universal mounting |
+| vmbdme | Dimmer for electronic/resistive load |
+| vmbdmi | Single channel triac dimmer for resistive and inductive loads |
+| vmbdmir | Single channel triac dimmer for resistive and inductive loads |
+| vmbel1 | Edge-lit one touch button module |
+| vmbel2 | Edge-lit two touch buttons module |
+| vmbel4 | Edge-lit four touch buttons module |
+| vmbelo | Edge-lit touch panel with Oled display |
+| vmbelpir | Edge-lit Motion detector with one touch button |
+| vmbgp1 | Glass control module with 1 touch key |
+| vmbgp1-2 | Glass control module with 1 touch key (Edition 2) |
+| vmbgp2 | Glass control module with 2 touch keys |
+| vmbgp2-2 | Glass control module with 2 touch keys (Edition 2) |
+| vmbgp4 | Glass control module with 4 touch keys |
+| vmbgp4-2 | Glass control module with 4 touch keys (Edition 2) |
+| vmbgp4pir | Glass control module with 4 touch keys and built-in motion and twilight sensor |
+| vmbgp4pir-2 | Glass control module with 4 touch keys and built-in motion and twilight sensor (Edition 2) |
+| vmbgpo | Glass control module with oled display |
+| vmbgpod | Glass control module with oled display and temperature controller |
+| vmbgpod-2 | Glass control module with oled display and temperature controller (Edition 2) |
+| vmbmeteo | Weather station with thermometer, anemometer, rain sensor and light sensor |
+| vmbpirc | Motion and twilight sensor for ceiling mounting |
+| vmbpirm | Mini motion and twilight sensor for recessed or surface mounting |
+| vmbpiro | Outdoor motion, twilight and temperature sensor, Theben |
+| vmbrfr8s | 8 channel RF receiver module |
+| vmbvp1 | Doorbird interface module |
+| vmbkp | Keypad interface module |
+| vmbin | 1-channel input module |
+| vmb4pb | 4 button interface module |
+
+The module type is the second segment of the unique thing ID. For example, a channel link to the
```

Comment:
The full structure is explained later on, so perhaps this example should be moved to there as well?